src/formatDate.js

```diff
@@ -50,7 +50,67 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let newFormatDate = '';
+
+  const oldDateSeparator = fromFormat[3];
+  const newDateSeparator = toFormat[3];
+  const oldDate = date.split(oldDateSeparator);
+  let oldYear;
+  const month = oldDate[fromFormat.indexOf('MM')];
+  const day = oldDate[fromFormat.indexOf('DD')];
+  let newYear;
+  const lastElementInNewDate = toFormat[2];
+
+  // finding year in old format date and converting it to new format
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments)